scout/parse/variant/transcript.py

```diff
@@ -200,9 +200,17 @@ def parse_transcripts(raw_transcripts, allele=None):
             LOG.debug("Something went wrong when parsing frequencies")
             LOG.debug("Only splitted and normalised VEP v90+ is supported")
 
+        if entry.get("CLINVAR_CLNVID"):
+            transcript["clinvar_clnvid"] = entry["CLINVAR_CLNVID"].split("&")
```

Comment:
Hm, I thought only CLNSIG was an &-separated list?

Reply:
I don't know. In theory it should just have one value here, but I don't know